.travis.yml

```diff
@@ -1,4 +1,3 @@
-if: (branch = master) OR (branch = development) OR (tag IS present)
```

Comment:
perhaps you can add this condition, instead of removing it.
`type = pull_request`

Reply:
@denisgranha @giacomolicari wdyt?

Reply:
@nicosampler i'm not sure whether including all the possible cases is better than removing this condition at all (which will cover all the cases by default)